mobile/lib/services/rest_api.dart

```diff
@@ -223,7 +223,7 @@ class AirqoApiClient {
 
       final response = await client.post(
         Uri.parse(
-          "${AirQoUrls.emailReAuthentication}?TOKEN=${Config.airqoApiV2Token}",
+          "${AirQoUrls.emailReAuthentication}/mobileAccountDelete?TOKEN=${Config.airqoApiV2Token}",
```

Comment:
thanks @BenjaminSsempala . 

On a side note I am yet to prove, I am not very sure if PROTECTED auth-service endpoints can be used without the JWT header. The route definitions (code) in the microservice could shed more light on this.

I look forward to your analysis on this. 

Thanks!

Reply:
Thanks @Baalmart , 
The JWT header is also added to the post request right below this line.

Reply:
thanks @BenjaminSsempala